crowbar_framework/app/models/node_object.rb

```diff
@@ -63,12 +63,17 @@ def self.find_node_by_alias(name)
   end
 
   def self.default_platform
-    @@default_platform ||= begin
-      admin = NodeObject.find("role:crowbar").select { |n| n.admin? }.first
-      if admin.nil?
-        ""
+    @default_platform ||= begin
+      provisioner = NodeObject.find("roles:provisioner-server").first
+      unless provisioner.nil? || provisioner["provisioner"]["default_os"].nil?
```

Comment:
Style/UnlessElse: Do not use `unless` with `else`. Rewrite these with the positive case first.